test/test_helper.rb

```diff
@@ -171,4 +171,13 @@ def admin_order_setup(status)
     fill_in 'Password', with: 'password'
     click_button 'Login'
   end
+
+  def create_featured_item
+    @featured_item = Item.create({ name: 'gnar possum',
+                                   description: 'a snowboard for shredding gnar',
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.